### PR TITLE
Resolves #2011

### LINKED ISSUE #2011: -AccessRight parameters incorrect for Server 2012 R2
[Enter feedback here]
The -AccessRight parameter says:
Specifies the access right to grant to the trustee. The acceptable values for this parameter are: Full, **Modify**, or Read.

Modify is not a valid parameter it should be Change (as it is on the Windows 10/2016 version of the doc. And in Example 2 on this page.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7e6d05fb-162b-a6e6-a090-71c142b27d09
* Version Independent ID: 5df21647-a94e-09f0-4264-c7de75da6ab3
* Content: [Grant-SmbShareAccess (smbshare)](https://docs.microsoft.com/en-us/powershell/module/smbshare/grant-smbshareaccess?view=winserver2012r2-ps)
* Content Source: [docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md)
* Product: **powershell**
* Technology: **powershell**
* GitHub Login: @andreabarr
* Microsoft Alias: **v-anbarr**

### PR DESCRIPTION
**Pull request by velkovb**
Fix AccessRight paremeter description.
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2011

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/smbshare/Grant-SmbShareAccess.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ The **Grant-SmbShareAccess** cmdlet adds an allow access control entry (ACE) for
 ## EXAMPLES
 
 ### Example 1: Adds an allow ACE for a trustee
```
Comment (JohanFreelancer9) [inline docset/windows/smbshare/Grant-SmbShareAccess.md:47]:
```suggestion
### Example 1: Adds an allows ACE for a trustee
```

File: docset/windows/smbshare/Grant-SmbShareAccess.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ VMFiles                 Contoso-SO              Contoso\Contoso-HV2$    Allow
 This command adds an allow ACE for a trustee to the security descriptor of the SMB share named VMFiles for the account named Contoso\Contoso-HV2$.
```
Comment (JohanFreelancer9) [inline docset/windows/smbshare/Grant-SmbShareAccess.md:62]:
```suggestion
This command adds an allows ACE for a trustee to the security descriptor of the SMB share named VMFiles for the account named Contoso\Contoso-HV2$.
```

File: docset/windows/smbshare/Grant-SmbShareAccess.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ VMFiles                 Contoso-SO              Contoso\Contoso-HV2$    Allow
...
 
 ### Example 2: Adds an allow ACE for a trustee without confirmation
```
Comment (JohanFreelancer9) [inline docset/windows/smbshare/Grant-SmbShareAccess.md:64]:
```suggestion
### Example 2: Adds an allows ACE for a trustee without confirmation
```

File: docset/winserver2012-ps/smbshare/Grant-SmbShareAccess.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ The **Grant-SmbShareAccess** cmdlet adds an allow access control entry (ACE) for
 ## EXAMPLES
 
 ### Example 1: Adds an allow ACE for a trustee
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/Grant-SmbShareAccess.md:38]:
```suggestion
### Example 1: Adds an allows ACE for a trustee
```

File: docset/winserver2012-ps/smbshare/Grant-SmbShareAccess.md [hunk 5]
```diff
@@ -53,7 +53,7 @@ VMFiles                 Contoso-SO              Contoso\Contoso-HV2$    Allow
 This command adds an allow ACE for a trustee to the security descriptor of the SMB share named VMFiles for the account named Contoso\Contoso-HV2$.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/Grant-SmbShareAccess.md:53]:
```suggestion
This command adds an allows ACE for a trustee to the security descriptor of the SMB share named VMFiles for the account named Contoso\Contoso-HV2$.
```

File: docset/winserver2012-ps/smbshare/Grant-SmbShareAccess.md [hunk 6]
```diff
@@ -53,7 +53,7 @@ VMFiles                 Contoso-SO              Contoso\Contoso-HV2$    Allow
...
 
 ### Example 2: Adds an allow ACE for a trustee without confirmation
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/smbshare/Grant-SmbShareAccess.md:55]:
```suggestion
### Example 2: Adds an allows ACE for a trustee without confirmation
```

File: docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md [hunk 7]
```diff
@@ -44,7 +44,7 @@ The **Grant-SmbShareAccess** cmdlet adds an allow access control entry (ACE) for
 ## EXAMPLES
 
 ### Example 1: Adds an allow ACE for a trustee
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md:46]:
```suggestion
### Example 1: Adds an allows ACE for a trustee
```

File: docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md [hunk 8]
```diff
@@ -61,7 +61,7 @@ VMFiles                 Contoso-SO              Contoso\Contoso-HV2$    Allow
 This command adds an allow ACE for a trustee to the security descriptor of the SMB share named VMFiles for the account named Contoso\Contoso-HV2$.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md:61]:
```suggestion
This command adds an allows ACE for a trustee to the security descriptor of the SMB share named VMFiles for the account named Contoso\Contoso-HV2$.
```

File: docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md [hunk 9]
```diff
@@ -61,7 +61,7 @@ VMFiles                 Contoso-SO              Contoso\Contoso-HV2$    Allow
...
 
 ### Example 2: Adds an allow ACE for a trustee without confirmation
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/smbshare/Grant-SmbShareAccess.md:63]:
```suggestion
### Example 2: Adds an allows ACE for a trustee without confirmation
```

File: docset/windows/smbshare/Grant-SmbShareAccess.md (hunk 1 above)
Comment (velkovb) [reply docset/windows/smbshare/Grant-SmbShareAccess.md:47]: It is allow access control entry (ACE). I believe allow is correct here.

**Comment by velkovb** [comment]
If there is a global effort maybe there is not point of doing it article by article.

**Comment by dariomws** [comment]
@velkovb I agree with you, I wrote to Chris to verify if we can do this global work. But I'm thinking that he'll approve this work in the Exchange repos, not in all repos (this one for example).

Let's see.

**Comment by e0i** [comment]
@dariomws @velkovb Let's keep the description for this one PR for now and I have included the AsJob as a mass-assignment into the backlog. See Teams channels for discussion. Thank you.

**Comment by sdwheeler** [comment]
> From Chris Davis:
> "AsJob is a common Windows PowerShell parameter that's available in remote PowerShell connections (similar to ErrorAction, ErrorVariable, etc.). platyPS was updated to not include that parameter (notice that it's not present in any Exchange-related cmdlets). I would consider a global search/replace effort to remove AsJob wherever you find it."
> 
> Maybe we should start removing it on any article we work on. What do you think?

`-AsJob` is not one of the Common Parameters in PowerShell. You should not remove it without verifying the accuracy. That parameter is cmdlet specific, not common.

The `-AsJob` parameter was removed from several cmdlet in PowerShell 6.0 due to the changes in remoting behavior.

```
# PowerShell 7
PS> Get-Command -ParameterName asjob

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Cmdlet          ForEach-Object                                     7.0.2.0    Microsoft.PowerShell.Core
Cmdlet          Invoke-Command                                     7.0.2.0    Microsoft.PowerShell.Core

#PowerShell 5.1
PS> Get-Command -ParameterName asjob

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Function        Disable-DscDebug                                   1.1        PSDesiredStateConfiguration
Function        Enable-DscDebug                                    1.1        PSDesiredStateConfiguration
Function        Get-DscConfiguration                               1.1        PSDesiredStateConfiguration
Function        Get-DscConfigurationStatus                         1.1        PSDesiredStateConfiguration
Function        Get-DscLocalConfigurationManager                   1.1        PSDesiredStateConfiguration
Function        Remove-DscConfigurationDocument                    1.1        PSDesiredStateConfiguration
Function        Restore-DscConfiguration                           1.1        PSDesiredStateConfiguration
Function        Stop-DscConfiguration                              1.1        PSDesiredStateConfiguration
Cmdlet          Get-WmiObject                                      3.1.0.0    Microsoft.PowerShell.Management
Cmdlet          Invoke-Command                                     3.0.0.0    Microsoft.PowerShell.Core
Cmdlet          Invoke-WmiMethod                                   3.1.0.0    Microsoft.PowerShell.Management
Cmdlet          Remove-WmiObject                                   3.1.0.0    Microsoft.PowerShell.Management
Cmdlet          Restart-Computer                                   3.1.0.0    Microsoft.PowerShell.Management
Cmdlet          Set-WmiInstance                                    3.1.0.0    Microsoft.PowerShell.Management
Cmdlet          Stop-Computer                                      3.1.0.0    Microsoft.PowerShell.Management
Cmdlet          Test-Connection                                    3.1.0.0    Microsoft.PowerShell.Management
Cmdlet          Test-DscConfiguration                              1.1        PSDesiredStateConfiguration
Workflow        Invoke-AsWorkflow                                  1.0.0.0    PSWorkflowUtility
```

**Pull request merged**